package.json

```diff
@@ -17,11 +17,17 @@
   "homepage": "https://videojs.com",
   "author": "Steve Heffernan",
   "scripts": {
```

Comment:
we're starting to get into fiddly territory with the amount of scripts here.

Reply:
Yeah, I think we need a pr once we are off of grunt to clean it all up.